patches/api/0114-Add-EntityKnockbackByEntityEvent.patch

```diff
@@ -65,8 +64,8 @@ index 0000000000000000000000000000000000000000..9efecabab813f575bb447a356e5e7e95
 +     */
 +    @NotNull
 +    @Override
-+    public LivingEntity getEntity() {
-+        return (LivingEntity) super.getEntity();
++    public Entity getEntity() {
++        return super.getEntity();
```

Comment:
You can't do this to the API. what if a plugin is using this event and is relying on this method returning a LivingEntity, as the contract currently states. but you change it to an entity (which I think is an ABI break), and suddenly the plugin is gonna have all sorts of problems.